### PR TITLE
created markdown file for python sets pop method

### LINKED ISSUE #4798: [Term Entry] Python Sets: `.pop()`
### Reason/inspiration (optional)

We would like a new term entry on the `.pop()` method for Sets in Python. The entry should go in a new file under `docs/content/python/concepts/sets/terms/pop/pop.md`

The entry should include the following:

- A description of the term
- A ##Syntax section that includes the syntax of the method and its details
- An ##Example section that includes a ```py ``` codeblock and ```shell ``` that shows the example output.
- A ##Codebyte Example to show the working of the method

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by evanshi0811**
### Description

Created a new markdown file for Python Sets .pop().

### Issue Solved

Closes #4798

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/python/concepts/sets/terms/pop/pop.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Here is one potential output of this program where `Beta` is randomly selected to be removed from `set`:
+
+```shell
+Beta
+{'Gamma', 'Alpha'}
+```
+
+## Codebyte Example (if applicable)
+
+In this codebyte example, the `.pop()` method randomly removes 2 elements from the set:
```
[the remark this answers is not in the document]
Comment (mamtawardhani) [reply content/python/concepts/sets/terms/pop/pop.md:48]: You can add a transitional statement like this for the codebyte.

File: content/python/concepts/sets/terms/pop/pop.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+```py
+set = {"Alpha", "Beta", "Gamma"}
+
+print(set.pop())
+
+print(set)
+```
+
+Here is one potential output of this program where `Beta` is randomly selected to be removed from `set`:
```
[the remark this answers is not in the document]
Comment (mamtawardhani) [reply content/python/concepts/sets/terms/pop/pop.md:39]: Specified that every time we will have a different output.

File: content/python/concepts/sets/terms/pop/pop.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  - 'Code Foundations'
+Tags:
+  - 'Collections'
+  - 'Functions'
+  - 'Methods'
+  - 'Sets'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
```
[the remark this answers is not in the document]
Comment (evanshi0811) [reply content/python/concepts/sets/terms/pop/pop.md]: Fixed in latest commit

File: content/python/concepts/sets/terms/pop/pop.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+In Python, the **`.pop()`** method randomly removes an element from a [set](https://www.codecademy.com/resources/docs/python/sets), and returns the element that was removed.
+
+> **Note:** If the set is empty, calling `.pop()` will cause an error.
```
[the remark this answers is not in the document]
Comment (evanshi0811) [reply content/python/concepts/sets/terms/pop/pop.md]: Fixed in latest commit

File: content/python/concepts/sets/terms/pop/pop.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+## Syntax
+
+```pseudo
+set.pop()
+```
+
```
[the remark this answers is not in the document]
Comment (evanshi0811) [reply content/python/concepts/sets/terms/pop/pop.md:27]: Added in latest commit

File: content/python/concepts/sets/terms/pop/pop.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (evanshi0811) [reply content/python/concepts/sets/terms/pop/pop.md]: Added in latest commit

File: content/python/concepts/sets/terms/pop/pop.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+Here is one potential output of this program where `Beta` is randomly selected to be removed from `set`:
+
+```shell
+Beta
+{'Gamma', 'Alpha'}
+```
+
+## Codebyte Example (if applicable)
```
[the remark this answers is not in the document]
Comment (evanshi0811) [reply content/python/concepts/sets/terms/pop/pop.md]: Fixed in latest commit

File: content/python/concepts/sets/terms/pop/pop.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (evanshi0811) [reply content/python/concepts/sets/terms/pop/pop.md]: Fixed in latest commit

**Comment by evanshi0811** [comment]
Uhh why are the checks failing now? I don't think I changed anything too drastically.

**Review by mamtawardhani — APPROVED** [review]
Thank you for the contribution @evanshi0811! Fixed the format check (There was an extra line in the entry) 😃 

The entry seems good for a second round of review!

**Review by Sriparno08 — APPROVED** [review]
Looks good, @evanshi0811! 👍

**Pull request merged**